woocommerce-myparcel.php

```diff
@@ -340,6 +340,10 @@ protected function upgrade($installed_version)
                 if (version_compare($installed_version, '4.2.1', '<=')) {
                     require_once('migration/wcmp-upgrade-migration-v4-2-1.php');
                 }
+
+                if (version_compare($installed_version, '4.3.2', '<=')) {
```

Comment:
Ik denk dat dit 4.4.1 moet worden. In develop staat nu namelijk 4.4.0 en deze aanpassing zal met de volgende release meegaan.

Reply:
Kijk ook op andere plekken waar je 4.3.2 heb genoteerd